[debug] [2019-02-10T14:00:25.578Z] ----------------------------------------------------------------------
[debug] [2019-02-10T14:00:25.580Z] Command:       /usr/local/bin/node /usr/local/bin/firebase serve
[debug] [2019-02-10T14:00:25.580Z] CLI Version:   6.3.1
[debug] [2019-02-10T14:00:25.581Z] Platform:      darwin
[debug] [2019-02-10T14:00:25.581Z] Node Version:  v10.13.0
[debug] [2019-02-10T14:00:25.581Z] Time:          Sun Feb 10 2019 08:00:25 GMT-0600 (Central Standard Time)
[debug] [2019-02-10T14:00:25.581Z] ----------------------------------------------------------------------
[debug] 
[debug] [2019-02-10T14:00:25.589Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2019-02-10T14:00:25.590Z] > authorizing via signed-in user
[debug] [2019-02-10T14:00:25.590Z] [iam] checking project checkup-15a61 for permissions ["firebase.projects.get"]
[debug] [2019-02-10T14:00:25.591Z] >>> HTTP REQUEST POST https://cloudresourcemanager.googleapis.com/v1/projects/checkup-15a61:testIamPermissions  

[debug] [2019-02-10T14:00:25.864Z] <<< HTTP RESPONSE 200
[debug] [2019-02-10T14:00:25.866Z] >>> HTTP REQUEST GET https://cloudresourcemanager.googleapis.com/v1/projects/checkup-15a61  
 
[debug] [2019-02-10T14:00:26.067Z] <<< HTTP RESPONSE 200
[info] 
[info] === Serving from '/Users/asiacarter/workspace/checkup'...
[info] 
[debug] [2019-02-10T14:00:26.527Z] Error: EACCES: permission denied, mkdir '/usr/local/lib/node_modules/firebase-tools/node_modules/@google-cloud/functions-emulator/logs'
    at Object.mkdirSync (fs.js:750:3)
    at Object.assertLogsPath (/usr/local/lib/node_modules/firebase-tools/node_modules/@google-cloud/functions-emulator/src/emulator/logs.js:65:10)
    at new Controller (/usr/local/lib/node_modules/firebase-tools/node_modules/@google-cloud/functions-emulator/src/cli/controller.js:77:32)
    at FunctionsEmulator.start (/usr/local/lib/node_modules/firebase-tools/lib/functionsEmulator.js:102:23)
    at Object.start (/usr/local/lib/node_modules/firebase-tools/lib/serve/functions.js:7:33)
    at /usr/local/lib/node_modules/firebase-tools/lib/serve/index.js:15:23
    at arrayMap (/usr/local/lib/node_modules/firebase-tools/node_modules/lodash/lodash.js:639:23)
    at Function.map (/usr/local/lib/node_modules/firebase-tools/node_modules/lodash/lodash.js:9556:14)
    at _serve (/usr/local/lib/node_modules/firebase-tools/lib/serve/index.js:13:26)
    at Command.module.exports.Command.description.option.option.option.option.before.action [as _action] (/usr/local/lib/node_modules/firebase-tools/lib/commands/serve.js:55:12)
[error] 
[error] Error: An unexpected error has occurred.
